packages/indexer-common/src/query-fees/object-conversion.ts

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash'
```

Comment:
We can remove this code, I think it's not being used.

Reply:
The whole file is only used on a test, we can remove that test because it's not how receipts are stored in the database anymore